packages/devtools_app/lib/src/version.dart

```diff
@@ -17,18 +17,34 @@ class FlutterVersion extends SemanticVersion {
     @required this.engineRevision,
     @required this.dartSdkVersion,
   }) {
+    final splitOnDash = version.split('-');
```

Comment:
comment what the format of a pre release version is expected to look like here.

Reply:
done